_minutes/2022-04-14-wecg.md

```diff
@@ -0,0 +1,142 @@
+ * [alexei] You're mentioning the possibility of there being two documents in the top-level frame, wouldn't that cause backwards compatibility issues?
+ * [dave] There is only one top level frame, index 0. A pre-render frame tree will be indicated by non-zero frameId - that's the backward incompatible part.
+ * [rob] To confirm, are you saying that the prerender frame will have a non-zero frameid, and that it's mutable? That sounds like a significant change in the extension API.
+ * [dave] In Chrome, frameId 0 is just an alias for the active frame. If something is tracking based on that frameId 0, that could be problematic.
+ * [rob] With the introduction of documentId, exposing this as a non-zero frame ID may be problematic.
+ * [dave] That's why we also include frameType, the top-level frame would have frameType “outermost_frame”.
+ * [tomislav] If we're introducing frameType to identify the top-level frame, then I would also be in favor of having a non-zero frameId everywhere.
+ * [oliver] Changing frameIds over the lifetime of the same document can cause issues, e.g. when runtime.onMessage changes frameId.
+ * [rob] given the concern over changing frame ID, how about leaving frameId as is and using documentId to target specific documents. If you are concerned about extensions not being prepared to deal with prerendered tabs, then you could consider allowing extensions to opt in to the visibility of prerendered tabs via the extension manifest.
+ * [dave] Prerendered has not shipped yet, we do not know yet how many extensions actually break.
+ * [timothy] I was of the impression that Chrome already had some prerendering support.
+ * [dave] Prerendering was shipped and unshipped on desktop. Our goal is that extensions will execute in prerender.
+ * [rob] Let's continue the rest of the discussion in the Github issue or in the Matrix chat since we've already spent half of the meeting on this.
+
+[Issue 191](https://github.com/w3c/webextensions/issues/191): Should background pages be allowed to navigate?
+
+ * [timothy] I filed this. There's some inconsistency across browsers. Safari allows this, Chrome blocks it, Firefox seems to allow it.
+ * [tomislav] Officially not by design in Firefox, we just didn't prevent it. Safe to say it's not something we intend to support.
+ * [rob] What about same origin navigation?
+ * [timothy] That's the only thing we allow in Safari.
+ * [rob] In the issue you mentioned “Safari 14+ allows navigations” - does that mean same-origin navigation only and not remote URLs?
+ * [timothy] Yes.
+ * [simeon] Already covered what Chrome does here, but
```

Comment:
@dotproto Did you have more thoughts that you said for this line?

Reply:
Thanks for tagging this, I'm updating this comment as part of my review.